Docs/contributing.md

wenderzxavier opened the pull request:
#11

Commit:
docs: adds CONTRIBUTING.md

Commit:
docs: update Readme.md with license and contributing

ZachPlunkett commented:
I know for myself I had to look up what to do once someone suggests changes on my pull request. I didn't know that I just push my new changes to the same Head I did before and that automatically updated the pull request.  

I don't know if that type of information is out of scope for this document.

ShaileshSurya commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,39 @@
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+6. Issue that pull request!
+
```
This is great. 

Can we add a sample comment message. Something like 
```
< Area of Code >: Short Description 

Long Description

Fixes : #Issue ID

author: username <email>
```

```
<Request builder>: Support for Request Context

Adding support for Go-Context pattern. 

Fixes: #1234

author : ShaileshSurya<ss.shailesh28@gmail.com>
```

ShaileshSurya reviewed (changes requested):
.

Commit:
<docs> : adds commit message format section

adds section on contributing.md file

fixes: #5

author: wender xavier <wender.zacariasxavier@gmail.com>

wenderzxavier reviewed (commented):
Added changes on the file :)

ShaileshSurya reviewed (approved):
Thank you @wenderzxavier  for your contribution. This indeed was needed for this library.
